random.go

```diff
@@ -0,0 +1,49 @@
+//
+// Copyright (c) 2018 HyperHQ.Inc
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package main
+
+import (
+	"io"
+	"os"
+	"syscall"
+)
+
+const (
+	rngDev     = "/dev/random"
+	rngDevMode = 0666
+
+	// include/uapi/linux/random.h
+	// RNDRESEEDCRNG   _IO( 'R', 0x07 )
+	_IOC_RNDRESEEDCRNG = 0x5207
+)
+
+func reseedRNG(data []byte) error {
+	if len(data) == 0 {
+		return nil
```

Comment:
The only legitimate reason for calling this function would be to call it with  some random data so I wonder if it's worth adding a log call for this scenario as a warning that something went wrong?

Reply:
hmm, good point! I'll fix it up. Thanks!